tests/unit/fp/fma-error.opt

```diff
@@ -0,0 +1,11 @@
+Name: target more poisonous than source
+; ERROR: Target is more poisonous than source for float %t
+%t = fma float %a, float %b, float %c
+  =>
+%t = poison
+
+Name: value mismatch (2.0 * 4.0) + 1.0 => 8.0
```

Comment:
due to limitations in the unit test runner, only 1 failing test per file is supported.

Reply:
Fixed, split into 2 files, fma-error-mismatch.opt and fma-poison-error.opt.